hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/RestoreJob.java

```diff
@@ -30,6 +30,9 @@
 
 @InterfaceAudience.Private
 public interface RestoreJob extends Configurable {
+
+  String KEEP_ORIGINAL_SPLITS_OPT = "hbase.backup.restorejob.keep_original_splits";
```

Comment:
Please place a constant with the default value for this config next to the key constant. That way there's one definition of default behavior that is easy to identify. As it is now, you have at least two `false` values that need to be kept in sync.